apps/ledger-live-desktop/src/renderer/families/solana/shared/fields/ValidatorsField.jsx

```diff
@@ -41,12 +43,16 @@ const ValidatorField = ({ t, account, onChangeValidator, chosenVoteAccAddr, stat
   }, [validators, chosenVoteAccAddr]);
 
   const validatorsFiltered = useMemo(() => {
-    return validators.filter(validator => {
-      return (
-        validator.name?.toLowerCase().includes(search) ||
-        validator.voteAccount.toLowerCase().includes(search)
+    return validators
+      .filter(validator => {
+        return (
+          validator.name?.toLowerCase().includes(search.toLowerCase()) ||
+          validator.voteAccount.toLowerCase().includes(search.toLowerCase())
+        );
+      })
+      .filter(
+        (value, index, self) => index === self.findIndex(t => t.voteAccount === value.voteAccount),
```

Comment:
feels like there is a faster algo to figure out. Here we are `O(n2)`

Reply:
<img width="981" alt="Screenshot 2022-07-12 at 13 30 01" src="https://user-images.githubusercontent.com/33897120/178480195-1cbb706b-7d9c-4fb5-8795-7844dcff424e.png">

Indeed as fun playing with the benchmark, thanks a lot !